src/main/org/nlogo/window/ButtonWidget.scala

```diff
@@ -374,15 +373,22 @@ class ButtonWidget(random:MersenneTwisterFast) extends JobWidget(random)
     def paintButtonRectangle(g: Graphics) {
       g.setColor(getPaintColor)
       g.fillRect(0, 0, getWidth(), getHeight())
-      def renderImages(g: Graphics, dark: Boolean) {
+      def renderImages(g: Graphics, inverse: Boolean) {
         def maybePaintForeverImage() {
           if (forever) {
-            val image = if (dark) FOREVER_GRAPHIC_DARK else FOREVER_GRAPHIC
+            val image = new ImageIcon(java.awt.Toolkit.getDefaultToolkit.createImage(
```

Comment:
I understand now.
Still, in the current state, having 4 images doubled doesn't seem right to me.
The pause and play can still be used instead of one forever icon, but it doesn't mean all other icons should be sort of duplicated.